plugins/rollbar-backend/src/api/RollbarApi.ts

```diff
@@ -104,7 +104,10 @@ export class RollbarApi {
     );
   }
 
-  private async get<T>(url: string, accessToken?: string) {
+  private async get<T extends readonly any[] | Record<string, any>>(
```

Comment:
I assume there probably was a type mismatch, can you hint me on why you had to change this (and the next function getForProject)?

Reply:
@aitherios I suggested this fix because the build fails when running `tsc:full`. The generic type `T` is passed down to the `camelcaseKeys` function, and it requires a type that extends a array or record, see the message below:
![image](https://user-images.githubusercontent.com/6290749/214011117-0169ea07-7944-4315-a613-dcee0ab88536.png)
@backstage/maintainers do you have any recommendations on how to fix the build error?

Reply:
@Rugvip also seems to be a breakage in a transitive upstream dependency.

Reply:
I'd say it's best to move this fix to a separate PR if it's still needed. We had a couple of upstream breakages the last week but they should all be patched by now.

In general it's best to keep these kind of fixes in their own PR since more people are likely affected and would want to ship it faster.